packages/@vue/cli-plugin-typescript/generator/convert.js

```diff
@@ -1,22 +1,33 @@
-module.exports = (api, { tsLint = false } = {}) => {
-  // delete all js files that have a ts file of the same name
-  // and simply rename other js files to ts
+module.exports = (api, { tsLint = false } = {}, convertAllFiles = true) => {
```

Comment:
What about merging the options into the second argument?

Reply:
Done! 👍